pkg/config/config.go

```diff
@@ -183,3 +191,8 @@ func DefaultConfig() PluginConfig {
 		},
 	}
 }
+
+const (
+	// RuntimeUntrusted is the implicit runtime defined for ContainerdConfig.UntrustedWorkloadRuntime
+	RuntimeUntrusted = "untrusted"
```

Comment:
Would be nice if we could move this string over to CRI maybe a TODO?

Reply:
true.. yet it would be beneficial to have example patterns and reserved names that have more meaning than .. anything :-)